app/src/main/java/org/schabi/newpipe/settings/VideoAudioSettingsFragment.java

```diff
@@ -58,10 +46,58 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
                             .show();
 
                 }
+            } else if (s.equals(getString(R.string.use_inexact_seek_key))) {
+                updateSeekOptions();
             }
         };
     }
 
+    /**
+     * Update fast-forward/-rewind seek duration options according to language and inexact seek setting.
+     * Exoplayer can't seek 5 seconds in audio when using inexact seek.
+     */
+    private void updateSeekOptions() {
+        //initializing R.array.seek_duration_description to display the translation of seconds
+        final Resources res = getResources();
+        final String[] durationsValues = res.getStringArray(R.array.seek_duration_value);
+        final List<String> displayedDurationValues = new LinkedList<>();
+        final List<String> displayedDescriptionValues = new LinkedList<>();
+        int currentDurationValue;
+        final boolean inexactSeek = getPreferenceManager().getSharedPreferences()
+            .getBoolean(res.getString(R.string.use_inexact_seek_key), false);
+
+        for (String durationsValue : durationsValues) {
+            currentDurationValue =
+                Integer.parseInt(durationsValue) / (int) DateUtils.SECOND_IN_MILLIS;
+            if (inexactSeek && currentDurationValue % 10 == 5) {
+                continue;
+            }
+            try {
+                displayedDurationValues.add(durationsValue);
```

Comment:
Even though this is not relative to this PR, I think this line should be before the try-catch, since it can't throw an exception, and for a moment it led me to thinking that for missing translations the value would not be added.
Also note the added newline.
```suggestion

            displayedDurationValues.add(durationsValue);
            try {
```